docs/docs/1.4.1/deployments/how-to-multi-zone.md

```diff
@@ -0,0 +1,311 @@
+      advertisedAddress: 10.0.0.1 # an address which other zones can use to consume this zone-ingress
+      advertisedPort: 10000 # a port which other zones can use to consume this zone-ingress" > ingress-dp.yaml
+    ```
+
+4. Apply the ingress config, passing the IP address of the remote control plane to `cp-address`:
+
+    ```
+    kuma-dp run \
+    --proxy-type=ingress \
+    --cp-address=https://<kuma-cp-address>:5678 \
+    --dataplane-token-file=/tmp/ingress-token \
+    --dataplane-file=ingress-dp.yaml
+    ```
+:::
+::::
+
+## Verify control plane connectivity
+
+You can run `kumactl get zones`, or check the list of zones in the web UI for the global control plane, to verify remote control plane connections.
+
+When a remote control plane connects to the global control plane, the `Zone` resource is created automatically in the global control plane.
+
+The Ingress tab of the web UI also lists remote control planes that you deployed with Ingress.
+
+## Set up cross-zone communication
+
+### Enable mTLS
+
+You must [enable mTLS](../policies/mutual-tls.md) for cross-zone communication between services.
+
+Kuma uses the Server Name Indication field, part of the TLS protocol, as a way to pass routing information cross zones. Thus, the mTLS is mandatory to enable cross-zone service communication.
+
+### Zone Ingress requirements
+
+Cross-zone communication between services is available only if Zone Ingress has a public address and public port.
+
+On Kubernetes, Kuma automatically tries to pick up the public address and port. Depending your load balancing implementation, you might need to wait a few minutes for Kuma to get the address.
+
+### Cross-communication details
+
+:::: tabs :options="{ useUrlFragment: false }"
+::: tab "Kubernetes"
+
+To view the list of service names available for cross-zone communication, run:
+
+```bash
+kubectl get dataplanes -n echo-example -o yaml | grep kuma.io/service
+           kuma.io/service: echo-server_echo-example_svc_1010
+```
+
+To consume the example service only within the same Kuma zone, you can run:
+
+```
+<kuma-enabled-pod>$ curl http://echo-server:1010
+```
+
+To consume the example service across all zones in your Kuma deployment (that is, from endpoints ultimately connecting to the same global control plane), you can run either of:
+
+```
+<kuma-enabled-pod>$ curl http://echo-server_echo-example_svc_1010.mesh:80
+<kuma-enabled-pod>$ curl http://echo-server.echo-example.svc.1010.mesh:80
+```
+
+And if your HTTP clients take the standard default port 80, you can the port value and run either of:
+
+```
+<kuma-enabled-pod>$ curl http://echo-server_echo-example_svc_1010.mesh
+<kuma-enabled-pod>$ curl http://echo-server.echo-example.svc.1010.mesh
+```
+
+Because Kuma on Kubernetes relies on transparent proxy, `kuma-dp` listens on port 80 for all virtual IPs that are assigned to services in the `.mesh` DNS zone. The DNS names are rendered RFC compatible by replacing underscores with dots.
```

Comment:
It might make sense to reference virtual-outbound to do more flexible things with hostnames and ports.